api/actions/loadArticles.js

```diff
@@ -14,8 +15,53 @@ export default function loadArticles(req, params = []) {
       if (err) {
         reject(err)
       } else {
-        resolve(res.body)
+        const queryObj = JSON.parse(_.get(query, 'embedded'))
+        let aricleRes = res.body
+        let writers =  []
+        const list = [ 'writters', 'photographers', 'designers', 'engineers' ]
+        list.forEach((item) => {
+          let aArr = _.get(aricleRes, item)
+          aArr.forEach((author) => {
+            if(author.image) {
+              writers.push('"' + author.image + '"')
```

Comment:
the backend api would return "Unable to parse `where` clause" if we don't use quotes